torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2507,6 +2507,9 @@ def large_1d_unique(dtype, device):
     samples.append(SampleInput(scalar, kwargs=dict(dim=0, descending=True, stable=True)))
     return samples
 
+def sample_inputs_argsort(*args, **kwargs):
+    return [sample_input for sample_input in sample_inputs_sort(*args, **kwargs) if "stable" not in sample_input.kwargs]
```

Comment:
This is a little lazy, but there is #57301 to unify the sample inputs of `sort` and `msort`, so maybe this could also be part of the refactor. cc @soulitzer